src/api/contacts/contacts.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+parent: API Reference
+title: Contact Details
+permalink: /api/contact-details
+---
+
+# Contact Details
+{:.no_toc}
+
+Contact Details represent information on how to contact a Centrapay user.
```

Comment:
I'm still not sure whether this should be Contact or Contact Details. There is some additional information to the contact details such as whether the phone number and email are verified.